### ISSUE TITLE
Linux: Python errors are prompted when launching the Asset Processor

### ISSUE THREAD

**Issue by LB-KacperKapusta** [o3de/o3de#5744]
**Describe the bug**
The following errors are displayed when launching the Asset Processor which has been built for the AutomatedTesting project:
```
Unable to load python library libpyside2.abi3.so.5.14 for Pyside2:
libpyside2.abi3.so.5.14: cannot open shared object file: No such file or directory
```
```
Unable to load python library libshiboken2.abi3.so.5.14 for Pyside2:
libshiboken2.abi3.so.5.14: cannot open shared object file: No such file or directory
```
Please note that after pressing 'OK' for both of the prompts, the assets for the project are processed.

**Prerequisites**
- AutomatedTesting is built for Linux using the instructions: https://o3de.org/docs/user-guide/platforms/linux/ 

**Steps to reproduce**
1. Launch the Asset Processor.
2. See the issue.

**Expected behavior**
Python errors are prompted when launching the Asset Processor.

**Actual behavior**
No errors are displayed when launching the Asset Processor.

**Screenshots/Video**
![AP_Python_error_1](https://user-images.githubusercontent.com/86953659/142418747-06ef5152-b449-4f9e-975b-3d26d926b334.png)
![AP_Python_error_2](https://user-images.githubusercontent.com/86953659/142418778-b6d40fea-4f42-4f2c-9b64-563979c8cd2a.png)

**Found in Branch**
Stabilization_2110 (d222aee)

**Desktop/Device**
Device: PC
OS: Linux
Version: Ubuntu 20.04.3 LTS
CPU: AMD Ryzen 5 3600
GPU: NVIDIA GeForce RTX 2060 SUPER
Memory: 16GB

**Additional context**
Could possibly be caused by the commit (https://github.com/o3de/o3de/commit/a051bdeeb0687d07851c0034fe9dc7e14dbf541f), which mentions the affected libraries.

**Comment by spham-amzn** [comment]
Fixed with https://github.com/o3de/o3de/pull/5773

**Comment by LB-KacperKapusta** [comment]
Verified the issue as fixed in Stabilization/2110 (ba32937).
